packages/shared/lib/utils/encryption.manager.ts

```diff
@@ -64,29 +64,14 @@ export class EncryptionManager extends Encryption {
         return decryptedEnvironment;
     }
 
-    public encryptApiConnection(connection: ApiConnection): Omit<StoredConnection, 'created_at' | 'updated_at'> {
```

Comment:
is `ApiConnection` still use elsewhere then? if no it can be deleted

Reply:
I have it cleaned already in a new PR it was just annoying to fix to have it deleted after 👌🏻